site-src/geps/gep-XXXX.md

```diff
@@ -0,0 +1,248 @@
+# GEP-XXXX: GRPCRoute
```

Comment:
When you next take a pass on this @gnossen, it's definitely worth logging an "Consider adding a GRPCRoute" issue with the `kind/gep` label, to get a number to put here. Even if we end up not doing this, having this GEP preserved as a record of the discussion is valuable.